### DISCUSSION TITLE
Searching Bar Feature

### DISCUSSION THREAD

**Discussion by woweya** [lawand-dot-io/taskbar#95] [Feature requests]
First of all, I would like to compliment you on the development of this Task Manager. My post is to suggest, as the title states, adding a "**Search Bar**" to be able to search for Files, Applications, and Folders from this bar (perhaps an icon that, when clicked, expands). Additionally, I would like to make a more aesthetic request: the possibility of increasing the **height of the Task Bar**, as personally, I find it a bit too "thin" and would prefer to make it larger. Maybe providing a range to make it bigger, but not excessively so.
Thanks for reading it!

**Comment by lawand-dot-io** [comment]
Hello. Thanks for the feedback. Those two feature requests already exist:
Search Bar: https://github.com/lawand-dot-io/taskbar/discussions/59
Taskbar size: https://github.com/lawand-dot-io/taskbar/discussions/36

Feel free to upvote them to increase the chance they get implemented. Thanks and enjoy Taskbar.
